src/index.js

```diff
@@ -0,0 +1,42 @@
+import * as domMan from './domManipulation';
+
+const checkWeather = (system, city) => {
+  fetch(`https://api.openweathermap.org/data/2.5/weather?q=${city}&units=${system}&appid=0747aa2b81c66ff632767f1576af3d12`, { mode: 'cors' })
```

Comment:
Opps, I guess you miss some important information again 😄 
One of the reasons for this project is to tests the knowledge of the student about asynchronous communication with promises or async/await and when to use them.

`Kindly make use of async-and-await`

Helpful Link: https://www.theodinproject.com/courses/javascript/lessons/async-and-await

Reply:
ok thanks i'll fix it